code/modules/surgery/organs/tongue.dm

```diff
@@ -330,7 +330,7 @@
 /obj/item/organ/tongue/slime
 	name = "slimey tongue"
 	desc = "It's a piece of slime, shaped like a tongue."
-	say_mod = list("blorbles", "bubbles")
+	say_mod = "blorbles"
```

Comment:
are you sure it is not supposed to be a list by default?

Reply:
All other tongues have a string, this one was a snowflake and Bacon said it's best to remove it.